src/App.tsx

```diff
@@ -1,223 +1,143 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, { useContext, useEffect, useState } from 'react';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { SortType } from './types/SortType';
+import { ErrorType } from './types/ErrorType';
+import { Todo } from './types/Todo';
+import { getActiveTodos, getCompletedTodos } from './utils/filtering';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   const user = useContext(AuthContext);
-  const newTodoField = useRef<HTMLInputElement>(null);
+
+  const [userTodos, setUserTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [sortType, setSortType] = useState<SortType>(SortType.all);
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.none);
```

Comment:
i think better save here error message

Reply:
Thanks for the review, i've fixed this in the next part of this task

Reply:
ok 👍